### PR TITLE
updating banner logic

### PR DESCRIPTION
**Pull request by kalexand-rh**
Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: _templates/_page_openshift.html.erb [hunk 1]
```diff
@@ -54,7 +54,7 @@
       </span>
       <% end %>
 
-      <% if (version == "4.13") && (distro_key != "openshift-webscale" || "openshift-dpu") %>
+      <% if (version == "4.13") && (distro_key != "openshift-webscale" && distro_key != "openshift-dpu") %>
```
Comment (aireilly) [inline _templates/_page_openshift.html.erb:57]:
```suggestion
<% if (version == "4.13") && (distro_key != "openshift-webscale" || distro_key != "openshift-dpu") %>
```

@gabriel-rh I think this is correct. WDYT

**Review by aireilly — COMMENTED** [review]
Small comment re: logic

File: _templates/_page_openshift.html.erb (hunk 1 above)
Comment (aireilly) [reply _templates/_page_openshift.html.erb:57]: IF 4.13 AND (openshift-webscale OR openshift-dpu)

**Review by gabriel-rh — APPROVED** [review]
/LGTM

**Pull request merged**